public/assets/js/account.js

```diff
@@ -7,15 +7,15 @@ editSettingsModalButtonClose?.addEventListener('click', () => {
 	editSettingsModal.classList.add('hidden');
 });
 
-document.addEventListener('click', event => {
+document.addEventListener('click', (event) => {
```

Comment:
Was this done manually or as part of eslint? I'm a little confused on this one since in this file the `()` are being *added*, but later on in the PR there's files where the `()` are being *removed*:

![Screenshot from 2025-01-21 12-18-14](https://github.com/user-attachments/assets/4bcf556a-234c-4cc9-9c43-99124438f2d2)

This feels inconsistent?

Reply:
This is all being done by eslint

I don't know the precise reason, but my guess is that single line expression lambdas don't have it, while multi line block lambdas do

Reply:
Just to follow up, this is the setting

It adds brackets as-needed (i.e. multiple parameters) but forces it for blocks

![image](https://github.com/user-attachments/assets/4553cfa4-26d7-43ba-a1e9-01348ada1842)